tests/desktop/consumer_pages/test_home_page.py

```diff
@@ -103,7 +102,6 @@ def test_settings_dropdown_menu(self, mozwebqa):
         user_settings = home_page.header.click_edit_account_settings()
         Assert.true(user_settings.is_the_current_page)
         Assert.true(user_settings.is_email_visible)
-        Assert.true(user_settings.is_email_non_editable)
```

Comment:
The email is not an input field anymore where we had a class == noneditable. The email is now just text.